frontend/src/lib/components/general/Header.svelte

```diff
@@ -70,7 +70,7 @@
 			{/if}
 		</div>
 		<div class="flex flex-col items-center justify-center mb-3">
-			{#if currentTab?.includes('project') && $project && $project.ownerName === user?.name}
+			{#if (currentTab?.includes('explore') || currentTab?.includes('compare')) && $project?.ownerName === user?.name}
```

Comment:
Don't we want to show this for chart? Seems like something people might want to access after chart creation.

Reply:
Good catch, it wasn't working at all before when checking for project.